metagraph/src/graph/alignment/aligner_seeder_methods.cpp

```diff
@@ -201,6 +220,11 @@ void SuffixSeeder<BaseSeeder>::generate_seeds() {
         size_t seed_length = 0;
         std::vector<node_index> alt_nodes;
 
+        if (!this->config_.no_seed_complexity_filter &&
```

Comment:
Why `no_`? Maybe it's better to call it `seed_complexity_filter` (though, the flag in CLI can still be `--no-...`)?

Reply:
I did it this way to make the variable have the same name as the CLI flag